src/components/SearchInput/SearchInput.constants.ts

```diff
@@ -1,12 +1,27 @@
+import { IconScale } from '../Icon';
+
 const CLASS_PREFIX = 'md-search-input';
 
-const DEFAULTS = {};
+const HEIGHTS = {
+  28: 28,
+  32: 32,
+};
```

Comment:
since the keys and values are the same, better if you are us an array instead 
```suggestion
const HEIGHTS = [28, 32] as const;
```

Note: the `as const` mean TS will infer `[28, 32]` type for the `HEIGHTS` variable instead `number[]`